rasa/train.py

```diff
@@ -206,21 +239,31 @@ def train_nlu(
 
     config = get_valid_config(config, CONFIG_MANDATORY_KEYS_NLU)
 
+    nlu_data_directory = data.get_nlu_directory(nlu_data)
+
+    if not os.listdir(nlu_data_directory):
+        print_error(
```

Comment:
I'm wondering whether it makes sense to do the validations in separate functions since they feel kind of duplicate, but somehow the error message is also always different 🤔